components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APIDetailedDTO.java

```diff
@@ -219,6 +221,32 @@ public void setApiLevelPolicy(String apiLevelPolicy) {
         this.apiLevelPolicy = apiLevelPolicy;
     }
 
+    /**
+     * * The authorization header of the API
+     **/
+    @Hash
+    @JsonProperty("authorizationHeader")
+    public String getAuthorizationHeader() {
+        return authorizationHeader;
+    }
+
+    public void setAuthorizationHeader(String authorizationHeader) {
+        this.authorizationHeader = authorizationHeader;
+    }
+
+
```

Comment:
Shall we remove this extra line? Also please check all the other places and do remove the extra lines.